src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -593,6 +593,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\CompilerServices\MethodImplAttribute.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\CompilerServices\MethodImplOptions.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\CompilerServices\IsReadOnlyAttribute.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\Runtime\CompilerServices\PreserveDependencyAttribute.cs" />
```

Comment:
This path needs to be fixed up to refer to the file at https://github.com/dotnet/runtime/blob/master/src/libraries/Common/src/System/Runtime/CompilerServices/PreserveDependencyAttribute.cs.  It just got moved in https://github.com/dotnet/runtime/pull/1307/files#diff-16e8cd74b890b37dd163d3a5921a35d5.

Reply:
Use `<Compile Include="$(CommonPath)...` to reference this file, and move it to the end of this block where other `CommonPath` references are.